### PR TITLE
Save chunks to rdb

### LINKED ISSUE #337: RDB store/load compressed data
Store data to disk in in compressed mode

### PR DESCRIPTION
**Pull request by ashtul**
fix #337

### CODE REVIEW

File: src/tsdb.h [hunk 1]
```diff
@@ -93,4 +94,26 @@ timestamp_t CalcWindowStart(timestamp_t timestamp, size_t window);
...
 
 CompactionRule *NewRule(RedisModuleString *destKey, int aggType, uint64_t timeBucket);
+
+typedef enum {
+    DICT_OP_SET = 0,
+    DICT_OP_REPLACE = 1,
+    DICT_OP_DEL = 2
+} DictOp;
+
+static inline int dictOperator(RedisModuleDict *d, void *chunk, timestamp_t ts, DictOp op) {
```
Comment (filipecosta90) [inline src/tsdb.h:104]: Typically, static functions are functions needed in only one file, meaning they are declared static to make that explicit by limiting their visibility. 

If the function is indeed used in multiple files with identical definitions it should be made external, with a single definition. If only one *.c file actually uses it, the declaration does not belong in a header.

**Review by filipecosta90 — CHANGES_REQUESTED** [review]

File: src/tsdb.h (hunk 1 above)
Comment (ashtul) [reply src/tsdb.h:104]: static in header file many times resolves multi definition issues ([example](https://stackoverflow.com/questions/47819719/static-inline-functions-in-a-header-file)).

**Review by filipecosta90 — APPROVED** [review]
LGTM @ashtul . Lets wait for @danni-m input also ( to be sure we don't miss a specific enterprise use case that it's not covered by the CI tests ).

File: src/tsdb.h (hunk 1 above)
Comment (ashtul) [reply src/tsdb.h]: change to only expose declaration in header...

**Pull request merged**